src/assets/css/admin-siwk.css

```diff
@@ -0,0 +1,8 @@
+tr:has(.siwk) {
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
```

Comment:
We should try to avoid important flags as much as possible. Can we be more specific in our selector at all to ensure we don't need it?

Reply:
Would the same selector with just the extra :has(.siwk) not be a more specific check in that case? We can keep this for now though, so no change needed. Just wanted to check!